libs/pascal_voc_io.py

```diff
@@ -141,10 +141,10 @@ def getShapes(self):
         return self.shapes
 
     def addShape(self, label, bndbox, difficult):
-        xmin = int(bndbox.find('xmin').text)
-        ymin = int(bndbox.find('ymin').text)
-        xmax = int(bndbox.find('xmax').text)
-        ymax = int(bndbox.find('ymax').text)
+        xmin = int(float(bndbox.find('xmin').text))
```

Comment:
why do you need to cast to float first, then cast to int?

Reply:
otherwise the parsing will fail - int can not parse a float but can convert a float value to int.

Reply:
This is what VoTT do.
```
 <object>
                                            <name>tallstock</name>
                                            <pose>Unspecified</pose>
                                            <truncated>0</truncated>
                                            <difficult>0</difficult>
                                            <bndbox>
                                                <xmin>4.861248761149653</xmin>
                                                <ymin>9.774427816901408</ymin>
                                                <xmax>3842.9600264288074</xmax>
                                                <ymax>314.5642605633803</ymax>
                                            </bndbox>
 </object>
```

Reply:
I happened to use VoTT for a few files but it was a bit buggy on OS-X so I turned back to labelimg but got annoyed by it not handling the VoTT files at all. But with this fix it will read them and show the boxes.

Reply:
Thanks for the explanation. Additionally, it will be better to have a unit test for it. If I have free time, I will add a unit test for that.

Reply:
I can send you a VoTT file if you like.